reddit_liveupdate/public/static/js/liveupdate/report.js

```diff
@@ -67,15 +67,34 @@
     },
 
     onAdminAction: function() {
-      var action = this.$adminButton.data('action')
-      var url = '/api/live/' + r.config.liveupdate_event + '/' + action
+      confirm_admin(this.$adminButton, r.config.liveupdate_event).then(function() {
+        window.location.reload();
+      });
+    },
+  });
 
-      r.ajax({
+  var confirm_admin = function($btn, thing_id) {
```

Comment:
`r.ui.ConfirmButton` triggers a `confirm` event on "yes".  It is fairly magical.  So how you feel about `post_admin_action`?